install/kubernetes/cilium/templates/cilium-agent/service.yaml

```
@@ -44,3 +45,24 @@ spec:
     targetPort: envoy-metrics
 {{- end }}
 {{- end }}
+{{- if and .Values.hubble.enabled .Values.hubble.listenAddress .Values.hubble.peerServicePort }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: hubble-peer
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: cilium
+spec:
+  selector:
+    k8s-app: cilium
+  ports:
+  - name: peer-service
+    port: {{ .Values.hubble.peerServicePort }}
+    protocol: TCP
+    targetPort: {{ (split ":" .Values.hubble.listenAddress)._1 }}
```

Let's handle IPv6 as well (e.g. `[::]:4244`)
```suggestion
    targetPort: {{ splitList ":" .Values.hubble.listenAddress | last }}
```